api/v1alpha1/etcd_types.go

```diff
@@ -230,6 +230,15 @@ type EtcdConfig struct {
 	// HeartbeatDuration defines the duration for members to send heartbeats. The default value is 10s.
 	// +optional
 	HeartbeatDuration *metav1.Duration `json:"heartbeatDuration,omitempty"`
+	// ClientService defines specified annotations that should be added to the client service
+	// +optional
+	ClientService *ClientServiceSpec `json:"clientService"`
+}
+
+type ClientServiceSpec struct {
```

Comment:
```suggestion
type ClientService struct {
```

Please also add a doc string.